### ISSUE TITLE
dnsbl query for IPv6

### ISSUE THREAD

**Issue by stanley101music** [dmippolitov/pydnsbl#31]
According to https://datatracker.ietf.org/doc/html/rfc5782#section-2.4 as mentioned in the project, the method for converting IPv6 address to IPv6 DNSxLs should be done by first expanding 0's -> reverse -> remove `:` -> concatenate each digit with `.`

For example
`2001:db8:1:2:3:4:567:89ab` -> `b.a.9.8.7.6.5.0.4.0.0.0.3.0.0.0.2.0.0.0.1.0.0.0.8.b.d.0.1.0.0.2`
Also need to consider the case where `::` exists
`2600:XXXX::f03c:91ff:fe50:d2` -> `2.d.0.0.0.5.e.f.f.f.1.9.c.3.0.f.0.0.0.0.0.0.0.0.X.X.X.X.0.0.6.2`

In line 165 of `pydnsbl/pydnsbl/checker.py` 
```
elif address.version == 6:
# according to RFC: https://tools.ietf.org/html/rfc5782#section-2.4
request_stripped = request.replace(':', '')
return '.'.join(reversed([x for x in request_stripped]))
```
It seems like the project didn't expand 0's first. Perhaps it should expand 0's first before continuing the rest steps.

**Comment by dmippolitov** [comment]
Hello, thanks. I will recheck this.

**Comment by dmippolitov** [comment]
fixed in 1.1.6
